kitsune/questions/views.py

```diff
@@ -1734,138 +1677,6 @@ def screen_share(request, question_id):
     )
 
 
-def _search_suggestions(request, text, locale, product_slugs):
```

Comment:
Let's leave this code for now with a comment to adapt it or merge it with featured articles.

Reply:
Why? Featured articles isn't ES driven, is it? And if we really need this code again we can find it in the git history.